### PR TITLE
[stable/traefik] Add File Backend

### PR DESCRIPTION
**Pull request by anhdat**
#### What this PR does / why we need it:

While Traefik's Kubernetes Ingress serves most common cases, there are advanced cases when the Kubernetes Backend is not capable of. This PR allows user to add File Backend which allows more fine grain control in the same Traefik instance. For example see this issue https://github.com/kubernetes/kubernetes/issues/41881

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by anhdat** [comment]
hi @mmatur @nmengin could you have a look at this PR? Thanks.

File: stable/traefik/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.69.1
+version: 1.69.2
```
Comment (SantoDE) [inline stable/traefik/Chart.yaml]: We add a new feature, so I guess we should do a minor version bump?

**Review by SantoDE — CHANGES_REQUESTED** [review]

File: stable/traefik/Chart.yaml (hunk 1 above)
Comment (anhdat) [reply stable/traefik/Chart.yaml]: I've bumped to 1.70.0

**Comment by anhdat** [comment]
@maorfr Thanks for reviewing. I've resolved the merge conflict on chart version. Please proceed.

**Pull request merged**